moped-editor/src/views/projects/projectView/ProjectComments.js

```diff
@@ -271,7 +270,6 @@ const ProjectComments = props => {
             <CommentButton noteTypeId={0}>All</CommentButton>
             <CommentButton noteTypeId={1}>Internal Notes</CommentButton>
             <CommentButton noteTypeId={2}>Status Updates</CommentButton>
-            <CommentButton noteTypeId={3}>Timeline Notes</CommentButton>
```

Comment:
I am wondering if there is code that requires this field to be on the page, or at least one with the `noteTypeId={3}`. I made a quick video to demonstrate it, and you can [find it here](https://austininnovation.slack.com/archives/CNUEPKLB1/p1648755790345859).  Thanks!